src/tests/specs/ink/Evaluation.spec.ts

```diff
@@ -23,7 +23,9 @@ describe("Evaluation", () => {
   // TestArithmetic
   it("tests arithmetic", () => {
     compileStory("arithmetic");
-    expect(context.story.ContinueMaximally()).toBe("36\n2\n3\n2\n2\n8\n8\n");
+    expect(context.story.ContinueMaximally()).toBe(
```

Comment:
I remember now why I paused the review the first time, I couldn't understand why inkjs was again able to differentiate between `1.0` & `1` … 🤦🏻‍♂️

Now I see it, it's because it doesn't go through the JSON intermediate representation here. We need to remember to revisit this test soon, so that we can assess the behaviour when loading a JSON file.